src/Microsoft.OData.Edm/EdmUtil.cs

```diff
@@ -231,7 +231,11 @@ internal static string ParameterizedName(IEdmOperation operation)
             foreach (IEdmOperationParameter parameter in operation.Parameters)
             {
                 string typeName = "";
-                if (parameter.Type.IsCollection())
+                if (parameter.Type == null)

```

Comment:
>parameter.Type == null [](start = 20, length = 22)

I cannot see how this typeName resolution for Untyped related to nav prop binding. Is this for a separate issue?

Reply:
Yes. I combined into same PR so we could verify in the same nightly builds.

---
In reply to: [175302690](https://github.com/OData/odata.net/pull/1109#discussion_r175302690) [](ancestors = 175302690)